cross/alsa-lib/Makefile

```diff
@@ -13,7 +13,7 @@ LICENSE  =
 
 GNU_CONFIGURE = 1
 CONFIGURE_ARGS = --disable-python
-ifeq ($(findstring $(ARCH),88f5281 ppc853x ppc854x),$(ARCH))
+ifeq ($(findstring $(ARCH),$(PPC_ARCHES)),$(ARCH))
```

Comment:
And line breaks in between. So, to be clear, the generic approach would look like this
```
LICENSE...

GNU_CONFIGURE ...if applicable

include ../...

CONFIGURE_ARGS = ...if applicable

ifeq($(findstring $(ARCH)...
  <thing>
endif

.PHONY etc. ...if applicable